apps/documentation/package.json

```diff
@@ -21,6 +22,9 @@
     "@radix-ui/react-separator": "^1.0.3",
     "@radix-ui/react-slot": "^1.0.2",
     "@t3-oss/env-nextjs": "^0.7.1",
+    "@tailwindcss/aspect-ratio": "^0.4.2",
```

Comment:
It would be great to find a way to keep these dependencies in the tailwind or ui packages, rather than bringing them in here.